<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog5</class>
 <widget class="QDialog" name="Dialog5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {

    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid#C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width:20ex;
    padding: 2px;
     font: bold, &quot;Comic Sans MS&quot;;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Commandes</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>2140</x>
       <y>1020</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius : 16px;
color: rgb(255, 255, 255);
background-color: rgb(0, 74, 108);</string>
     </property>
     <item>
      <property name="text">
       <string>TRADUCTION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>francais</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>anglais</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>1660</x>
       <y>590</y>
       <width>581</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Afficher les employés</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>1840</x>
       <y>880</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Modifier ou Supprimer un produit</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>990</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Rechercher les produit</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Par prix&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>72</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Par Qte&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>159</x>
        <y>82</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>118</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Par Libellé&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>128</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>2220</x>
       <y>880</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Extaire Fichier PDF</string>
     </property>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Extraire</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>1740</x>
       <y>990</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Trier les produits</string>
     </property>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Par prix décroissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Par prix croissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Par libellé</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>2330</x>
       <y>540</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../HelloWorld/qt/recource.qrc">:/Artboard-35-ConvertImage.png</pixmap>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>517</x>
       <y>50</y>
       <width>581</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Afficher les commandes</string>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>551</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>340</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Modifier ou Supprimer une commandes</string>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>450</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Rechercher les commandes</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Par prix&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>1077</x>
       <y>340</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Extaire Fichier PDF</string>
     </property>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Extraire/Imprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_14">
     <property name="geometry">
      <rect>
       <x>597</x>
       <y>450</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Trier les commandes</string>
     </property>
     <widget class="QPushButton" name="pushButton_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Par prix décroissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Par prix croissant</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>0</y>
       <width>1281</width>
       <height>631</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../HelloWorld/qt/recource.qrc">:/15190.jpg</pixmap>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #00002d;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Ajouter une commandes</string>
     </property>
     <widget class="QPushButton" name="pushButton_VAL_AJ_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_QUIT_AJ_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CODE_C">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CODE_C</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;TYPE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ID_C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;DATE_C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;PRIX_TOT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TYPE">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_C">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DATE_C">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRIX_TOT">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>1187</x>
       <y>0</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../HelloWorld/qt/recource.qrc">:/Artboard-35-ConvertImage.png</pixmap>
     </property>
    </widget>
    <zorder>label_14</zorder>
    <zorder>comboBox</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>groupBox_12</zorder>
    <zorder>groupBox_4</zorder>
    <zorder>groupBox_11</zorder>
    <zorder>label_8</zorder>
    <zorder>groupBox_5</zorder>
    <zorder>groupBox_6</zorder>
    <zorder>groupBox_13</zorder>
    <zorder>groupBox_7</zorder>
    <zorder>groupBox_14</zorder>
    <zorder>groupBox_8</zorder>
    <zorder>label_20</zorder>
   </widget>
   <widget class="QWidget" name="s">
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
